### DISCUSSION TITLE
How to link two PBPK models in PK-Sim?

### DISCUSSION THREAD

**Discussion by Gautamvj** [Open-Systems-Pharmacology/Forum#1227] [How-Tos] [answered]
Hello Folks,
Myself Gautam Vijaywargi.

I am new to the use of PK-Sim software and trying to learn it using examples. While doing so, I came across an example where I need to establish a link between two PBPK models using one parameter. The first model is of a parent moiety which upon metabolism yields the product moiety. The conversion between them is guided by a rate constant and I don't know exactly how to link the validated PBPK models of both the moieties via this rate constant using PK-Sim. Do I have to use MoBi in this case or PK-Sim itself is sufficient to take care of this? I have created both the compounds in a single PK-Sim project. 

Thank you.

**Comment by StephanSchaller** [comment, marked as answer]
When creating an Enzame reaction, you have to specify the metabolite here:
![image](https://user-images.githubusercontent.com/25477643/173525652-887f4121-017c-4347-b5f5-9b0a71bcbcba.png)


See also the Documentation:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#definition-of-a-metabolite-in-an-enzymatic-process

**Reply by Gautamvj** [reply]
Thank you Stephen, this sure has helped me and I was able to link my PBPK models.
